src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -51,7 +51,11 @@ export const KEYBINDING_CONTEXT_TERMINAL_PROCESS_SUPPORTED = new RawContextKey<b
 export const IS_WORKSPACE_SHELL_ALLOWED_STORAGE_KEY = 'terminal.integrated.isWorkspaceShellAllowed';
 export const NEVER_MEASURE_RENDER_TIME_STORAGE_KEY = 'terminal.integrated.neverMeasureRenderTime';
 export const SHOW_TERMINAL_CONFIG_PROMPT = 'terminal.integrated.showTerminalConfigPrompt';
-export const HAS_TYPED_IN_TERMINAL = 'terminal.integrated.hasTypedInput';
+
+export const EXCLUDED_KEYS = ['RightArrow', 'LeftArrow', 'UpArrow', 'DownArrow', 'Space', 'Meta', 'Control', 'Shift', 'Alt', '', 'Delete', 'Backspace', 'Tab'];
```

Comment:
I would move these two consts to be closer to the code that uses them. They don't seem like generic terminal concepts to me. Like it's not clear what "excluded" without knowing the context. Everything else looks good!